### PR TITLE
Ely Hess

### PR DESCRIPTION
**Pull request by elyhess**

### CODE REVIEW

**Comment by elyhess** [comment]
I just want to add, so that it's on record. I completely overcomplicated this. I was closer to the answer in my original attempt, but the commit history does not show all the various things I tried. Ultimately I got hung up on the `write` method, as I was using `assert_instance_of Book, jane_eyre.class` which led me on several tangents ending up trying to use .self and ultimately running out of time. I'll get em next time.

**Comment by KathleenScriver** [comment]
I'd love to see more commits. A commit when you've written a test, then another commit when you have the code written to make that test pass.

**Comment by KathleenScriver** [comment]
Thanks for that comment! Us devs are good at over-complicating things. :)

File: lib/book.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :title, :author, :publication_year
+
+  def initialize(data)
+    @title = data[:title]
+    @author = data[:author_first_name] + " " + data[:author_last_name]
+    @publication_year = data[:publication_date][-4..-1]
```
Comment (KathleenScriver) [inline lib/book.rb:7]: I'd rather keep the whole date here, and then make a method that just pulls the year, that way I have access to more data if I need it.

**Review by KathleenScriver — COMMENTED** [review]
Score was a 1.5. I encourage you to reclone this and go through it again for practice and repetition. You got this!

**Pull request closed without merging**